web/packages/design/src/RadioGroup/RadioGroup.tsx

```diff
@@ -46,20 +49,30 @@ export function RadioGroup({
   onChange,
   autoFocus,
   name,
+  size,
   ...styles
 }: RadioGroupProps) {
   return (
-    <Flex flexDirection="column" {...styles}>
+    <Flex gap={3} flexDirection="column" {...styles}>
       {options.map((option, index) => {
         const optionValue = isRadioObjectOption(option) ? option.value : option;
+        const optionLabel = isRadioObjectOption(option) ? option.label : option;
+        const optionDisabled = isRadioObjectOption(option)
+          ? option.disabled
+          : undefined;
         return (
-          <Radio
-            onChange={onChange}
-            autoFocus={index === 0 && autoFocus}
+          <FieldRadio
             key={optionValue}
-            option={option}
             name={name}
+            label={optionLabel}
+            //helperText={...}
```

Comment:
```suggestion
```

Reply:
Damn, you actually pointed out a small feature that I forgot to implement! :) I'm gonna do it in another PR, I guess.